packages/react-composites/tests/browser/call/app/index.tsx

```diff
@@ -1,58 +1,44 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { AzureCommunicationTokenCredential, CommunicationUserIdentifier } from '@azure/communication-common';
-import React, { useState, useEffect } from 'react';
+import React from 'react';
 import ReactDOM from 'react-dom';
 
 import { _IdentifierProvider } from '@internal/react-components';
-import {
-  CallAdapter,
-  CallAdapterState,
-  createAzureCommunicationCallAdapter,
-  CallComposite,
-  COMPOSITE_LOCALE_FR_FR,
-  COMPOSITE_LOCALE_EN_US
-} from '../../../../src';
+import { CallAdapterState, CallComposite, COMPOSITE_LOCALE_FR_FR, COMPOSITE_LOCALE_EN_US } from '../../../../src';
 import { IDS } from '../../common/constants';
-import { isMobile, verifyParamExists } from '../../common/testAppUtils';
+import { isMobile } from '../../common/testAppUtils';
 import memoizeOne from 'memoize-one';
 import { IContextualMenuItem, mergeStyles } from '@fluentui/react';
-import { fromFlatCommunicationIdentifier } from '@internal/acs-ui-common';
+import { MockCallAdapter } from './MockCallAdapter';
 
 const urlSearchParams = new URLSearchParams(window.location.search);
 const params = Object.fromEntries(urlSearchParams.entries());
 
-// Required params
-const displayName = verifyParamExists(params.displayName, 'displayName');
-const token = verifyParamExists(params.token, 'token');
-const groupId = verifyParamExists(params.groupId, 'groupId');
-const userId = verifyParamExists(params.userId, 'userId');
+const state = JSON.parse(params.state) as CallAdapterState;
 
 // Optional params
 const useFrLocale = Boolean(params.useFrLocale);
 const showCallDescription = Boolean(params.showCallDescription);
 const injectParticipantMenuItems = Boolean(params.injectParticipantMenuItems);
 
-function App(): JSX.Element {
```

Comment:
There would be another app such that the other snapshot tests are not affected. This is rough POC.